peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -4,45 +4,47 @@
 {% block title %}{{ document.title }}{% endblock %}
 
 {% block page-content %}
-  <main class="pb-5">
-    <div class="container">
-      {% block breadcrumbs %}
-      {% endblock %}
+  <div class="pb-5">
```

Comment:
page-content was already wrapped in a `main` element tag, so we had a nested `main`, generally isnt semantically correct